lib/cstruct.ml

```diff
@@ -339,12 +339,15 @@ let fillv ~src ~dst =
   aux dst 0 src
 
 
-let to_string t =
+let to_bytes t =
   let sz = len t in
   let b = Bytes.create sz in
   unsafe_blit_bigstring_to_bytes t.buffer t.off b 0 sz;
+  b
+
+let to_string t =
   (* The following call is safe, since b is not visible elsewhere. *)
```

Comment:
`since b is not visible elsewhere` ↝ `since this is the only reference to the freshly-created value built by [to_bytes t]`.